frontend/i18next.scanner/pt/translation.json

```diff
@@ -514,7 +514,7 @@
   "Error while saving new news": "Erro ao salvar a nova notícia",
   " is calling you on Tracim": " está a chamar-lhe no Tracim",
   "{{author}} called you": "{{author}} chamou-te",
-  "<0> <1>{{userCalleeName}}</1> has accepted your call. If the call has not opened click on this<3> link </3></0> ": "<0> <1>{{ userCalleeName }}</1> aceitou a sua chamada. Se a chamada não tiver sido aberta, clique neste<3> link </3></0>",
+  "<0> <1>{{userCalleeName}}</1> has accepted your call. If the call has not opened click on this<3> link </3></0> ": "<0> <1>{{ userCalleeName }}</1> aceitou a sua chamada. Se a chamada não tiver sido aberta, clique neste<3>link</3></0> ",
```

Comment:
The old translation seems better (spaces are more coherent with the key)

Reply:
But is missing the final space, I'll merge both translations.